to-do-list/src/components/PostForm/PostForm.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {useState, useCallback} from 'react'
+import MyInput from '../UI/input/MyInput'
+import { PostItemProps } from '../Interfaces/interfaces'
+import {PostFormProps} from '../Interfaces/interfaces'
+import FormFooter from './FormFooter'
+import CategoryOptions from './CategoryOptions'
+import FormHeader from './FormHeader'
+
+const PostForm = ({create, setVisible} : PostFormProps) =>{
+    const [post, setPost] = useState<PostItemProps>({title:'', dateValue:'', completed: false})
```

Comment:
It's better to split it to different states and create object on send to process values separately